azext_iot/tests/central/test_iot_central_devices_int.py

```diff
@@ -99,9 +99,12 @@ def test_central_device_twin_show_success(self):
             checks=[self.check("deviceId", device_id)],
         )
         self._delete_device(device_id=device_id, api_version=self._api_version)
-        self._delete_device_template(
-            template_id=template_id, api_version=self._api_version
-        )
+        try:
+            self._delete_device_template(
+                template_id=template_id, api_version=self._api_version
```

Comment:
If there is an issue with deleting device templates, then hide the error? What about introducing a polling element to ensure devices associated with a target template are first deleted?

Reply:
assuming we always operate with CLI-born device templates, we can delete devices with no worries.
I followed your suggestion and delete associated devices for the template. INT tests screenshots updated